Entity search in verejne changed from button to input

PeterPsota opened the pull request:
The input no longer behaves like a button and opens the modal only on form submit.
(The first commit is already a part of PR #94)

borisvida commented on client/src/components/Verejne/index.js:
```diff
@@ -26,13 +32,23 @@ const Verejne = ({
+            type="text"
+            placeholder={FIND_ENTITY_TITLE}
+            value={entitySearchValue}
+            onChange={(e) => setEntitySearchValue(e.target.value)}
```
Please, move this to `withHandlers`, because that arrow causes unnecessary re-renders of `Input` component.

borisvida commented on client/src/components/Verejne/EntitySearch/index.js:
```diff
@@ -64,8 +65,8 @@ const EntitySearch = ({
-              value={searchEntityValue}
-              onChange={(e) => setSearchEntityValue(e.target.value)}
+              value={entitySearchValue}
+              onChange={(e) => setEntitySearchValue(e.target.value)}
```
Maybe I already wrote it to another PR with this commit, but please, move this to `withHandlers` too.

borisvida commented on client/src/components/Verejne/EntitySearch/index.js:
```diff
@@ -90,15 +91,16 @@ const EntitySearch = ({
+    {setEntitySearchValue, toggleModalOpen, setEntitySearchFor}
   ),
-  withState('searchEntityValue', 'setSearchEntityValue', ''),
+  withState(''),
```
Does this line make any sense anymore?

PeterPsota commented:
@borisvida I've moved the arrow functions and removed the pointless withState.
Should I update the other PR as well, or do we close it and proceed with just this one?

PeterPsota commented:
Fixed merge conflicts, pls merge
